servers/audio/effects/audio_effect_limiter.cpp

```diff
@@ -29,38 +29,66 @@
 /*************************************************************************/
 
 #include "audio_effect_limiter.h"
+#include "core/math/math_funcs.h"
 
+/*Soft clipping starts to reduce the peaks a little below the threshold level
+ *and progressively increases its effect as the input level increases such that
+ *the threshold is never exceeded.
+*/
+
+//I'm implementing this as a logistics function because it's designed to increase near
+//linearly and taper up to a certain max value, which is exactly what we want for
+//this soft clipping function
```

Comment:
Follow the existing comment code style in place:

```suggestion
// Soft clipping starts to reduce the peaks a little below the threshold level
// and progressively increases its effect as the input level increases such that
// the threshold is never exceeded.
//
// This is implemented as a logistics function because it's designed to increase near
// linearly and taper up to a certain max value. This is exactly what we want for
// this soft clipping function.
```